deploy/app/main.tf

```diff
@@ -30,12 +32,100 @@ provider "aws" {
   }
 }
 
-# access state for shared resources (primary DNS zone, dev VPC and dev caches)
-data "terraform_remote_state" "shared" {
-  backend = "s3"
-  config = {
-    bucket = "storacha-terraform-state"
-    key    = "storacha/indexing-service/shared.tfstate"
-    region = "us-west-2"
+# CloudFront is a global service. Certs must be created in us-east-1, where the core ACM infra lives
```

Comment:
> Certs must be created in us-east-1

"global"

Reply:
this is storoku generated code and should be fixed there